homework-16/server/src/test.rs

```diff
@@ -0,0 +1,55 @@
+#[cfg(test)]
+mod tests {
```

Comment:
You don't need to have two nested mods like this, you can stick the cfg on the short `mod test` line in the parent module